### PR TITLE
Replace Quotes

### PR DESCRIPTION
**Pull request by soerenmartius**
Everything in #3 and:

- chore: replace double with single quotes in index.js files

### CODE REVIEW

File: packages/eslint-config-basic/index.js [hunk 1]
```diff
@@ -4,133 +4,190 @@ module.exports = {
...
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+    'indent': ['error', 2, { SwitchCase: 1, VariableDeclarator: 1, outerIIFEBody: 1 }],
+    'no-restricted-syntax': [
+      'error',
+      'DebuggerStatement',
+      'ForInStatement',
+      'LabeledStatement',
+      'WithStatement',
     ],
-    "no-spaced-func": "error",
```
Comment (mariux) [inline packages/eslint-config-basic/index.js:56]: why was this removed?

**Review by mariux — CHANGES_REQUESTED** [review]
really hard to review as changing quotes and content at the same time..

in addition resoning for changing various values is missing... what are the changes doing and why?

File: packages/eslint-config-basic/index.js (hunk 1 above)
Comment (soerenmartius) [reply packages/eslint-config-basic/index.js:56]: > really hard to review as changing quotes and content at the same time..
> 
> in addition resoning for changing various values is missing... what are the changes doing and why?

well, you should review #3 first as written in the description of this PR.

Comment (soerenmartius) [reply packages/eslint-config-basic/index.js:56]: that's why https://github.com/mineiros-io/eslint-config/pull/3 is opened as a separate PR - it provides all the changes separately from this PR that only changes quotes
